lib/foreman_theme_satellite/documentation.rb

```diff
@@ -78,7 +78,7 @@ module Documentation
         'Products_and_Repositories_content-management' => "#{ForemanThemeSatellite.documentation_root}/managing_content/importing_content_content-management#Products_and_Repositories_content-management",
       },
       'Managing_Hosts' => {
-        'registering-a-host_managing-hosts' => "#{ForemanThemeSatellite.documentation_root}/managing_hosts/registering_hosts_to_server_managing-hosts#Registering_Hosts_by_Using_Global_Registration_managing-hosts",
+        'registering-a-host_managing-hosts' => "#{ForemanThemeSatellite.documentation_root}/managing_hosts/registering-hosts-and-setting-up-host-integration_managing-hosts#Registering_Hosts_by_Using_Global_Registration_managing-hosts",
```

Comment:
@Lennonka I have rolled back the change to the key (`registering-a-host_managing-hosts`). This key is used in the upstream foreman. We want to make sure that when the code calls for the upstream value like here:
https://github.com/theforeman/foreman/blob/9d0f388296a845ebcba944e9fc04570a42f5ceea/webpack/assets/javascripts/react_app/routes/RegistrationCommands/RegistrationCommandsPage/index.js#L203
the theme will serve the downstream link.